apps/balancer-tools/src/app/apr/round/[roundId]/page.tsx

```diff
@@ -12,6 +13,12 @@ export default function Page({
 }: {
   params: { roundId: string };
 }) {
+  const { setSelectedPool } = useSelectedPoolRoundContext();
```

Comment:
wouldn't we already be setting the selected pool in the path? How an additional context would be helpful here?

Reply:
you mean instead of using `setSelectedPool` we could just redirect to the pool url?

Reply:
yup @luizakp